app/src/main/res/values/plurals.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <plurals name="num_of_directory_items">
+    <item quantity="one">1 item</item>
```

Comment:
Should still be `%1$s item` so it's properly localised if need be.

Reply:
Done!